templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -183,6 +183,10 @@ if [[ "$TEST" == "performance" ]]; then
   exit
 fi
 
+# Run proxy.py
+podman run -d -it -p 8899:8899 --name ciproxy --rm ghcr.io/abhinavsingh/proxy.py:latest --basic-auth foo:bar --hostname 0.0.0.0
```

Comment:
can you add that container in the same way as we start the minio container?

Reply:
I can see this being overengineered, but i'd like to see the CI containers to be started in one place.